fastify.js

```diff
@@ -93,7 +131,7 @@ function build () {
       const handle = node[req.method]
 
       if (handle && req.method === 'GET') {
-        handleNode(handle, params, req, res, null)
+        handleNode(handle, params, req, res, urlUtil.parse(req.url, true).query)
```

Comment:
We can validate querystrings _before_ calling `handleNode`, and before parsing the body. At the end, the querystring needs to go into `Request.query`.